src/locales/fr-FR/admin.ftl

```diff
@@ -399,7 +399,7 @@ decisionHistory-goToComment = Aller au commentaire
 ## moderate
 moderate-navigation-reported = Signalé
 moderate-navigation-pending = En attente
-moderate-navigation-unmoderated = Non modéré
+moderate-navigation-unmoderated = Réactions
```

Comment:
this tab does not show all comments, only those which _have not_ been moderated.

Reply:
Ok but we have to find something better that "non modéré" because in french it doesn't really mean anything significant. I will think about that.

Reply:
Do you have any more ideas on how to translated "Unmoderated" @grfa54? Maybe "Untouched" "To Moderate", something along those lines?

Reply:
it is complicated. The "non mondéré" term in french implies that the comments were not validated by a human **but that they should**. Like "You need to see thoses unmoderated comments because they contains profanity". That's why I used the terme "Reactions" to tell the user that thoses are "just comments"